### PR TITLE
Improved is_displayed error handle

### PR DESCRIPTION
**Pull request by boludoz**

### CODE REVIEW

**Comment by boludoz** [comment]
The code will fail if you cannot handle that error, I recommend returning false.

**Comment by boludoz** [comment]
@kaliiiiiiiiii Improved error handling a bit, since that could be the cause of the error.

**Comment by boludoz** [comment]
image
On the rebound I have solved a bug in click, since in reality the error has a dot at the end, using 'in' is already solved.

**Comment by kaliiiiiiiiii** [comment]
> image I also improved the others a little using in.

@boludoz Can you provide an example where that would make a difference?

File: src/selenium_driverless/types/webelement.py [hunk 1]
```diff
@@ -632,9 +632,15 @@ async def shadow_root(self):
...
-        # Only go into this conditional for browsers that don't use the atom themselves
-        size = await self.size
-        return not (size["height"] == 0 or size["width"] == 0)
+        try:
+            # Only go into this conditional for browsers that don't use the atom themselves
+            size = await self.size
+            return not (size["height"] == 0 or size["width"] == 0)
+        except CDPError as e:
+            if e.code == -32000 and 'Could not compute box model.' in e.message:
+                return False
```
Comment (kaliiiiiiiiii) [inline src/selenium_driverless/types/webelement.py:641]: Should raise an exception in my view.
Returning `False` just isn't something a user would expect, I suppose

**Review by kaliiiiiiiiii — CHANGES_REQUESTED** [review]

**Comment by kaliiiiiiiiii** [comment]
> The code will fail if you cannot handle that error, I recommend returning false.

@boludoz 
Should raise an exception in my view.
Returning `False` just isn't something a user would expect, I suppose

**Review by kaliiiiiiiiii — APPROVED** [review]
nvm you're right😂

**Review by kaliiiiiiiiii — APPROVED** [review]

**Pull request merged**